canonical/library.canonical.com
# Add unquote line to link parser

## Done

- Decodes ASCII as part of the link parser

## QA

- Go to https://library-canonical-com-58.demos.haus/tests-and-issues/copy-of-raise-a-jira-ticket-for-sales-ops and click the link for 'Jira ticket'
- It should retain the query parameters in the url
